app/scripts/app.js

```diff
@@ -631,6 +631,13 @@ angular
       // content (e.g. using :before pseudo-elements).
       $('body').addClass('ios');
     }
+  })
+  .run(function($rootScope, APIService) {
+    var resources = APIService.availableKinds();
+    var mobileClient = _.find(resources, function(resource) {
```

Comment:
This can be just

```js
$rootScope.AEROGEAR_MOBILE_ENABLED = _.some(resources, { kind: 'MobileClient' });
```

But I'd recommend

```js
$rootScope.AEROGEAR_MOBILE_ENABLED = !!APIService.apiInfo({ group: '<your-group>', kind: 'MobileClient' });
```

That way you don't have to loop over all kinds and you check specifically for your API group (whatever it is).